lookout/style/typos/analyzer.py

```diff
@@ -40,50 +38,7 @@
 
 IDENTIFIER = bblfsh.role_id("IDENTIFIER")
 IMPORT = bblfsh.role_id("IMPORT")
-
-
-class IdTyposModel(AnalyzerModel):
-    """
-    A model to store approved identifiers from the repo.
-    """
-
-    LICENSE = DEFAULT_LICENSE
-
-    def __init__(self, **kwargs):
-        """Construct a FormatModel."""
-        super().__init__(**kwargs)
-        self._identifiers = set()
-
-    def update(self, new_identifiers: Iterable[str]) -> None:
-        """
-        Add new identifiers to the set of approved identifiers.
-
-        :param new_identifiers: New identifiers to add to the set of approved identifiers.
-        """
-        self._identifiers.update(new_identifiers)
-
-    def dump(self) -> str:
-        """Serialize this model and return the result as a string."""
-        return super().dump() + \
-            "\nNumber of known and approved identifiers is %s" % len(self._identifiers)
-
-    def _generate_tree(self) -> dict:
-        tree = super()._generate_tree()
-        tree.update(identifiers=merge_strings(self._identifiers))
-        return tree
-
-    def _load_tree(self, tree: dict) -> None:
-        super()._load_tree(tree)
-        self._identifiers = split_strings(tree["identifiers"])
-
-    def __len__(self) -> int:
-        return len(self._identifiers)
-
-    def __iter__(self):
-        yield from self._identifiers.__iter__()
-
-    def __contains__(self, identifier: str) -> bool:
-        return identifier in self._identifiers
+IDENTIFIER_ID_COLUMN = "identifier_id"
```

Comment:
If you unbox the abbreviation, it means "identifier_identifer". Let's rename to "identifier_index", if it **is** an index